cfg/config.yaml

```diff
@@ -147,6 +147,7 @@ node:
       - "/etc/kubernetes/kubelet/kubelet-config.json"
       - "/home/kubernetes/kubelet-config.yaml"
       - "/home/kubernetes/kubelet-config.yml"
+      - "/etc/default/kubeletconfig.json"
```

Comment:
FYI: the order of these files matters, so here we expect to check with `/etc/default/kubeletconfig.json` first, then `/etc/default/kubetlet`, since both files present in AKS cluster. But I am not 100% sure this is the correct way to do this.

Reply:
Since the is no way to priorities beside tops to bottom this is the correct way, I actually didn't find any information online beside the link you posted in github, but there is no harm in adding new paths :) after the build will run I will approve it